quarry/data/packets.csv

```diff
@@ -80,7 +80,8 @@ minecraft_version,protocol_version,protocol_mode,packet_direction,packet_name
 1.12.1,338,play,downstream,advancements
 1.12.1,338,play,downstream,entity_effect
 1.12.1,338,play,upstream,teleport_confirm
-1.12.1,338,play,upstream,prepare_crafting_grid
+1.12.1,338,play,upstream,craft_recipe_request
+1.12.1,338,play,downstream,craft_recipe_response
 1.12.1,338,play,upstream,tab_complete

```

Comment:
Can you shed some light on this? I did push the change, but I'm curious if it's a packet order thing or a preference thing.